src/app/components/annotations/TiplineRequest.js

```diff
@@ -147,26 +148,30 @@ const TiplineRequest = ({
         intl.formatMessage(messages.smoochNoMessage)
       )}
       icon={<SmoochIcon name={messageType} />}
-      historyButton={
-        <TiplineHistoryButton
+      historyButton={(
+        showButtons && <TiplineHistoryButton
           uid={uid}
           name={userName}
           channel={channelLabel[messageType] || messageType}
           messageId={messageId}
         />
-      }
-      sendMessageButton={
-        <SendTiplineMessage
+      )}
+      sendMessageButton={(
+        showButtons && <SendTiplineMessage
           username={userName}
           channel={channelLabel[messageType] || messageType}
           annotationId={activity.dbid}
         />
-      }
+      )}
       receipt={<RequestReceipt events={reportHistory} />}
     />
   );
 };
 
+TiplineRequest.defaultProps = {
+  showButtons: true,
```

Comment:
I think boolean defaults should always be false. This way we can simply use the prop name when true or omit it as false, instead of passing propName={false}

Reply:
This is a good point. We could change the name of the variable to `hideButtons` and have it be `false` by default. It's a double negative but I think it's more react-idiomatic.

Reply:
Good idea, guys, thanks. I changed it to `hideButtons`.